_pages/playbook/principles.md

```diff
@@ -1,166 +1,4 @@
 ---
-title: meta.principles.title
 permalink: /playbook/principles/
-description: meta.principles.description
-class: relative
+redirect_to: /
```

Comment:
I think we can remove this entire file if we add `redirect_from: /playbook/principles` to the root page, same for the other files that link back to existing pages in this site

Reply:
I went ahead and pushed a change that did this in d9435d0efb3af7465ee1ece19424033ab56b8930

Reply:
Great. I just added the trailing `/` at the end of the partners links in [2ddb95f](https://github.com/18F/identity-site/pull/483/commits/2ddb95f44831df6a95c6608b05bd6fe39f7eb1f1) Looks like `make test-urls` won't pick it up without it?